### PR TITLE
Allow disabling limits after deployment

### PR DESCRIPTION
**Pull request by karlb**
We want to be able to remove the limits without redeploying the contracts. We do this by setting the limits to MAX_INT.

Closes https://github.com/raiden-network/raiden-contracts/issues/1481.

To safely allow 3rd parties to create token networks after the limit removal, creating new TNs must not involve any parametrization. This is done by forcing the limits to be MAX_INT after the limits have been removed from the TNR.

Closes https://github.com/raiden-network/raiden-contracts/issues/1416.

### CODE REVIEW

**Review by palango — COMMENTED** [review]
Looks good in general, nice small commits make this easy to review. After seeing the additions of `changeController`, I think we should add a base class `Controllable` (or some other name) to bundle the ownership related functionality.

Also in the last commit you should update the gas costs file.

File: raiden_contracts/data/source/raiden/TokenNetwork.sol [hunk 1]
```diff
@@ -11,6 +11,8 @@ import "raiden/Token.sol";
 import "raiden/Utils.sol";
 import "raiden/SecretRegistry.sol";
 
+uint256 constant MAX_INT = 2**256 - 1;
```
[the remark this answers is not in the document]
Comment (karlb) [reply raiden_contracts/data/source/raiden/TokenNetwork.sol]: I was irritated by the name (what makes this number safe?) and didn't spot that `115792089237316195423570985008687907853269984665640564039457584007913129639935 == 2**256 -1`. I'll remove the duplication and change the definitions of MAX_SAFE_UINT256 to `2**256 -1` to make it's meaning more obvious.

**Review by palango — APPROVED** [review]
Great!

**Comment by palango** [comment]
We need to add a changelog entry, but this can be done in a different PR.

**Pull request merged**